tools/peview/main.c

```diff
@@ -117,13 +117,18 @@ INT WINAPI wWinMain(
 #ifndef DEBUG
     if (PhIsExecutingInWow64())
     {
-        PhShowWarning(
-            NULL,
+        PhGuiSupportInitialization();
+        PhSettingsInitialization();
+        PvInitializeSettings();
+        PvInitializeSuperclassControls();
+        PhShowWarning2(
+            GetDesktopWindow(),
```

Comment:
I'm seeing this frequently in the PR, why the changes to pass the desktop window handle in these paths? It seems unnecessary given the documentation and SAL.

Reply:
Recently @dmex also aked me about this. At your discretion, I can revert this.

> I changed parrent from NULL to GetDesktopWindow() to not show double icon on this dialogs. It wasn't a problem for MessageBox, since it didn't have window icon.
> 
> Parrent `NULL`
> ![image](https://github.com/user-attachments/assets/7ca12402-0eba-4db4-8e9c-d40eee233181)
> Parrent `Desktop`
> ![image](https://github.com/user-attachments/assets/90eb778b-9509-4882-8098-ecdcb590c06a)